### ISSUE TITLE
Spicetify currently does not support Spotify `1.1.85`

### ISSUE THREAD

**Issue by Daksh777** [spicetify/spicetify-cli#1660]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: Spotify for Windows 1.1.85.895.g2a71e1b8
- Spicetify version: 2.9.7
```

### 📝 Description

On applying any theme, I see a blank/black screen

**Comment by R1911** [comment]
also happens on 2.9.8

**Comment by xerta555** [comment]
Here we go again for 1.1.85.895.g2a71e1b8 so 🙃

**Comment by JonahBerr** [comment]
i have same issue but no blank screen, i just updated spotify but when i run spicetify apply it shows this: 

Applying additional modifications:
error cannot find symbol for Custom app React Element
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/spicetify/spicetify-cli/src/apply.insertCustomApp.func1({0x140008b4000, 0x18aa98})
	/private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/apply/apply.go:220 +0x708
github.com/spicetify/spicetify-cli/src/utils.ModifyFile({0x140000a7580, 0x3e}, 0x1400019b8c8)
	/private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/utils/utils.go:191 +0x80
github.com/spicetify/spicetify-cli/src/apply.insertCustomApp({0x140000a7580?, 0x0?}, {{0x14000162500, 0x7, 0x8}, {0x140000a02c0, 0x4, 0x4}, 0x1, 0x1, ...})
	/private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/apply/apply.go:188 +0x90
github.com/spicetify/spicetify-cli/src/apply.AdditionalOptions({0x1400001c400, 0x31}, {{0x14000162500, 0x7, 0x8}, {0x140000a02c0, 0x4, 0x4}, 0x1, 0x1, ...})
	/private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/apply/apply.go:37 +0x3ec
github.com/spicetify/spicetify-cli/src/cmd.Apply({0x100b8f3a4, 0x5})
	/private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/cmd/apply.go:72 +0x864
main.main()
	/private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/spicetify.go:225 +0x7b8

spotify still works fine but it doesnt have any of the modifications from spicetify

**Comment by ssh-sysadmin** [comment]
After updating spotify, ran `spicetify auto`, and now it does not load. It's stock spotify now.
```
C:\Users\Admin>spicetify auto
spicetify v2.9.8
info There is available backup.
info Clear current backup:
success Backup is cleared.
Backing up app files:
OK
Extracting:
OK
Preprocessing:
info Fetching remote CSS map for newer compatible tag version: master
OK
OK
success Everything is ready, you can start applying now!
Copying raw assets:
OK
Transferring user.css:
OK
Applying additional modifications:
error cannot find symbol for Custom app React Element
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/spicetify/spicetify-cli/src/apply.insertCustomApp.func1({0xc00072c000, 0x18aa98})
        /home/runner/work/spicetify-cli/spicetify-cli/src/apply/apply.go:220 +0x987
github.com/spicetify/spicetify-cli/src/utils.ModifyFile({0xc000010480, 0x38}, 0xc0002af868)
        /home/runner/work/spicetify-cli/spicetify-cli/src/utils/utils.go:191 +0x7b
github.com/spicetify/spicetify-cli/src/apply.insertCustomApp({0xc000010480, 0x3b}, {{0x146cff8, 0x0, 0x0}, {0xc000aa0000, 0x2, 0x2}, 0x1, 0x1, ...})
        /home/runner/work/spicetify-cli/spicetify-cli/src/apply/apply.go:188 +0x90
github.com/spicetify/spicetify-cli/src/apply.AdditionalOptions({0xc00000eab0, 0x2b}, {{0x146cff8, 0x0, 0x0}, {0xc000aa0000, 0x2, 0x2}, 0x1, 0x1, ...})
        /home/runner/work/spicetify-cli/spicetify-cli/src/apply/apply.go:37 +0x4c3
github.com/spicetify/spicetify-cli/src/cmd.Apply({0x123b7f8, 0x5})
        /home/runner/work/spicetify-cli/spicetify-cli/src/cmd/apply.go:72 +0x9ae
github.com/spicetify/spicetify-cli/src/cmd.Auto({0x123b7f8, 0x5})
        /home/runner/work/spicetify-cli/spicetify-cli/src/cmd/auto.go:32 +0x1d5
main.main()
        /home/runner/work/spicetify-cli/spicetify-cli/spicetify.go:247 +0x72f

```

**Comment by MonsteRico** [comment]
Having same exact issue with the runtime rror

**Comment by CharlieS1103** [comment]
Patched the first broken hook, one more and we should be all good.

**Comment by SaberAkiyama** [comment]
respond @itsmeow

**Comment by itsmeow** [comment]
I think Charlie has it covered? If I'm needed to fix something I can.

**Comment by SaberAkiyama** [comment]
Sometimes the OG Spotify has different codes or added for it.

**Comment by Clcq1** [comment]
> After updating spotify, ran `spicetify auto`, and now it does not load. It's stock spotify now.
> 
> ```
> C:\Users\Admin>spicetify auto
> spicetify v2.9.8
> info There is available backup.
> info Clear current backup:
> success Backup is cleared.
> Backing up app files:
> OK
> Extracting:
> OK
> Preprocessing:
> info Fetching remote CSS map for newer compatible tag version: master
> OK
> OK
> success Everything is ready, you can start applying now!
> Copying raw assets:
> OK
> Transferring user.css:
> OK
> Applying additional modifications:
> error cannot find symbol for Custom app React Element
> panic: runtime error: index out of range [0] with length 0
> 
> goroutine 1 [running]:
> github.com/spicetify/spicetify-cli/src/apply.insertCustomApp.func1({0xc00072c000, 0x18aa98})
>         /home/runner/work/spicetify-cli/spicetify-cli/src/apply/apply.go:220 +0x987
> github.com/spicetify/spicetify-cli/src/utils.ModifyFile({0xc000010480, 0x38}, 0xc0002af868)
>         /home/runner/work/spicetify-cli/spicetify-cli/src/utils/utils.go:191 +0x7b
> github.com/spicetify/spicetify-cli/src/apply.insertCustomApp({0xc000010480, 0x3b}, {{0x146cff8, 0x0, 0x0}, {0xc000aa0000, 0x2, 0x2}, 0x1, 0x1, ...})
>         /home/runner/work/spicetify-cli/spicetify-cli/src/apply/apply.go:188 +0x90
> github.com/spicetify/spicetify-cli/src/apply.AdditionalOptions({0xc00000eab0, 0x2b}, {{0x146cff8, 0x0, 0x0}, {0xc000aa0000, 0x2, 0x2}, 0x1, 0x1, ...})
>         /home/runner/work/spicetify-cli/spicetify-cli/src/apply/apply.go:37 +0x4c3
> github.com/spicetify/spicetify-cli/src/cmd.Apply({0x123b7f8, 0x5})
>         /home/runner/work/spicetify-cli/spicetify-cli/src/cmd/apply.go:72 +0x9ae
> github.com/spicetify/spicetify-cli/src/cmd.Auto({0x123b7f8, 0x5})
>         /home/runner/work/spicetify-cli/spicetify-cli/src/cmd/auto.go:32 +0x1d5
> main.main()
>         /home/runner/work/spicetify-cli/spicetify-cli/spicetify.go:247 +0x72f
> ```

same issue as this, any fix created yet?

**Comment by IrepY** [comment]
Same issue, any fix yet?

**Comment by StarNumber12046** [comment]
same problem

**Comment by Kafva** [comment]
Also getting `error cannot find symbol for Custom app React Element` on macOS

**Comment by Levkia14** [comment]
Same issue, any fix yet?

**Comment by kaiyin2** [comment]
Getting the same error message right after Spotify client **updated to the version 1.1.85.895.g2a71e1b8**

![image](https://user-images.githubusercontent.com/67120455/168392511-017d456a-980e-487b-863d-df5eed2177d9.png)

**Comment by Gamershy** [comment]
Go figure something like this breaks when I first install the program. I thought it was something I did because it was working just fine with the windows store version lmao.

**Comment by RenRenIsWeird** [comment]
> i have same issue but no blank screen, i just updated spotify but when i run spicetify apply it shows this:
> 
> Applying additional modifications: error cannot find symbol for Custom app React Element panic: runtime error: index out of range [0] with length 0
> 
> goroutine 1 [running]: github.com/spicetify/spicetify-cli/src/apply.insertCustomApp.func1({0x140008b4000, 0x18aa98}) /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/apply/apply.go:220 +0x708 github.com/spicetify/spicetify-cli/src/utils.ModifyFile({0x140000a7580, 0x3e}, 0x1400019b8c8) /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/utils/utils.go:191 +0x80 github.com/spicetify/spicetify-cli/src/apply.insertCustomApp({0x140000a7580?, 0x0?}, {{0x14000162500, 0x7, 0x8}, {0x140000a02c0, 0x4, 0x4}, 0x1, 0x1, ...}) /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/apply/apply.go:188 +0x90 github.com/spicetify/spicetify-cli/src/apply.AdditionalOptions({0x1400001c400, 0x31}, {{0x14000162500, 0x7, 0x8}, {0x140000a02c0, 0x4, 0x4}, 0x1, 0x1, ...}) /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/apply/apply.go:37 +0x3ec github.com/spicetify/spicetify-cli/src/cmd.Apply({0x100b8f3a4, 0x5}) /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/cmd/apply.go:72 +0x864 main.main() /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/spicetify.go:225 +0x7b8
> 
> spotify still works fine but it doesnt have any of the modifications from spicetify

I have same error as this

**Comment by ririxidev** [comment]
It's known. We are working on fix.

**Comment by Luc-onGitHub** [comment]
Hey, I wanted to mention that I have the same error/issue. Would be happy when this issue is resolved.

**Comment by Serdna2203** [comment]
Same problem whit the issue

**Comment by Im-Kal** [comment]
# Fix for #1660:
- Go to your spicetify installation folder and delete "Backup"
- Reinstall Spotify using this [link](https://upgrade.scdn.co/upgrade/client/win32-x86/spotify_installer-1.1.84.716.gc5f8b819-8.exe)
- Relogin and run ```spicetify backup apply```
- Success!

**Comment by ririxidev** [comment]
that's not fix, it's workaround by installing older version of spotify for windows users.

**Comment by Im-Kal** [comment]
Alright, I'll reflect the same.

**Comment by Ivan9962** [comment]
> # Temporary workaround for #1660:
> * Go to your `.spicetify` folder and delete "Backup"
> * Reinstall Spotify using this [link](https://upgrade.scdn.co/upgrade/client/win32-x86/spotify_installer-1.1.84.716.gc5f8b819-8.exe)
> * Relogin and run `spicetify backup apply`
> * Success!

I tried this and now is it working without any problem!

**Comment by Im-Kal** [comment]
> > # Temporary workaround for #1660:
> > 
> > * Go to your `.spicetify` folder and delete "Backup"
> > * Reinstall Spotify using this [link](https://upgrade.scdn.co/upgrade/client/win32-x86/spotify_installer-1.1.84.716.gc5f8b819-8.exe)
> > * Relogin and run `spicetify backup apply`
> > * Success!
> 
> I tried this and now is it working without any problem!

Great to hear, be sure to update Spotify and Spicetify when the fix is pushed out.

**Comment by raffy309** [comment]
> i have same issue but no blank screen, i just updated spotify but when i run spicetify apply it shows this:
> 
> Applying additional modifications: error cannot find symbol for Custom app React Element panic: runtime error: index out of range [0] with length 0
> 
> goroutine 1 [running]: github.com/spicetify/spicetify-cli/src/apply.insertCustomApp.func1({0x140008b4000, 0x18aa98}) /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/apply/apply.go:220 +0x708 github.com/spicetify/spicetify-cli/src/utils.ModifyFile({0x140000a7580, 0x3e}, 0x1400019b8c8) /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/utils/utils.go:191 +0x80 github.com/spicetify/spicetify-cli/src/apply.insertCustomApp({0x140000a7580?, 0x0?}, {{0x14000162500, 0x7, 0x8}, {0x140000a02c0, 0x4, 0x4}, 0x1, 0x1, ...}) /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/apply/apply.go:188 +0x90 github.com/spicetify/spicetify-cli/src/apply.AdditionalOptions({0x1400001c400, 0x31}, {{0x14000162500, 0x7, 0x8}, {0x140000a02c0, 0x4, 0x4}, 0x1, 0x1, ...}) /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/apply/apply.go:37 +0x3ec github.com/spicetify/spicetify-cli/src/cmd.Apply({0x100b8f3a4, 0x5}) /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/src/cmd/apply.go:72 +0x864 main.main() /private/tmp/spicetify-cli-20220506-84532-1s98dx8/spicetify-cli-2.9.8/dep/spicetify.go:225 +0x7b8
> 
> spotify still works fine but it doesnt have any of the modifications from spicetify

i have the same issue an di have tried anything i know to fix it and i can't. Probably the new spotify update messed with the code an dnow we can't apply/inject spicetify in it. IDK

**Comment by itsmeow** [comment]
My apologies, I haven't had the time to work on a fix.

**Comment by CharlieS1103** [comment]
Patched in v2.10
